grunt/config/browserify.js

```diff
@@ -206,7 +206,7 @@ var addonsMin = {
 // The DOM Builds
 var dom = {
   entries: [
-    './build/node_modules/react-dom/lib/ReactDOMUMDEntry.js',
+    './build/node_modules/react-dom/lib/ReactDOMFiber.js',
```

Comment:
Sounds reasonable.

I'm a little confused by the module resolution system. Does `require("ReactDOM")` in UMDEntery points to the specific file or to the whole ReactDOM package (which is already exporting Fiber)?

The latter would mean that UMD build featured Fiber before this change. Which is false. Ok I got it.